contracts/acl/ACL.sol

```diff
@@ -109,8 +109,9 @@ contract ACL is IACL, TimeHelpers, AragonApp, ACLHelpers {
     */
     function grantPermission(address _entity, address _app, bytes32 _role)
         external
+        onlyPermissionManager(_app, _role)
```

Comment:
Is this supposed to be an optimization? I don’t see the benefit in terms of gas usage per call, while I guess the bytecode will increase (for instance, modifiers are inlined).
Besides, if we do this, I would either make the `internal` call `private` or mark it as “unsafe” as we are not checking the `onlyPermissionManager` condition.

Reply:
Yea, we can move the modifier to the `internal` method then